tests/tracking/test_rest_tracking.py

```diff
@@ -12,31 +12,18 @@
 import time
 import tempfile
 
-from click.testing import CliRunner
 
 import mlflow.experiments
 from mlflow.entities import RunStatus, Metric, Param, RunTag
 from mlflow.protos.service_pb2 import LOCAL as SOURCE_TYPE_LOCAL
-from mlflow.server import app, BACKEND_STORE_URI_ENV_VAR
-from mlflow.store.file_store import FileStore
+from mlflow.server import app, BACKEND_STORE_URI_ENV_VAR, ARTIFACT_ROOT_ENV_VAR
 from mlflow.tracking import MlflowClient
+from mlflow.tracking.utils import _tracking_store_registry
 from mlflow.utils.mlflow_tags import MLFLOW_RUN_NAME, MLFLOW_PARENT_RUN_ID, MLFLOW_SOURCE_TYPE, \
     MLFLOW_SOURCE_NAME, MLFLOW_PROJECT_ENTRY_POINT, MLFLOW_GIT_COMMIT
-
+from tests.integration.utils import invoke_cli_runner
```

Comment:
Use this helper function to run CLI commands instead of CLIRunner - the helper just wraps CLIRunner & verifies the CLI command exited with code 0 (previously we didn't verify the exit code of CLI commands)